client/mobrender/components/mobilization.js

```diff
@@ -78,7 +75,11 @@ class Mobilization extends React.Component {
                 href='http://www.bonde.org/'
                 className='right my2'
                 target='_blank'>
```

Comment:
Now that we have the `exenv` lib installed, may be a good idea to handle the load of icon only on clientside..